internal/common/pagination/entitlement.go

```diff
@@ -23,6 +23,7 @@ var entitlementResources = []openfga.Kind{
 	openfga.ModelType,
 	openfga.ApplicationOfferType,
 	openfga.GroupType,
+	openfga.RoleType,
```

Comment:
i wonder how the tests were passing before

Reply:
Maybe no roles?